### PR TITLE
feat(plugin-meetings): events for media:serverNegotiatedwith experimental.enableMediaNegotiatedEvent config

### PR DESCRIPTION
**Pull request by arun3528**
Added event for CNN to handle the transfer scenario

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

**Review by jonniespratley — APPROVED** [review]

**Review by taymoork2 — APPROVED** [review]
LGTM

File: packages/node_modules/@webex/plugin-meetings/src/roap/handler.js [hunk 1]
```diff
@@ -34,7 +34,7 @@ const checkForAndHandleErrors = (action, meeting, correlationId) => {
   return false;
 };
 
-const compareRemoteRoapOffer = (lastRoapMessage, currentRoapMessage) => currentRoapMessage.messageType === _OFFER_ && lastRoapMessage?.msg?.seq === currentRoapMessage.msg.seq;
+const compareRemoteRoapOffer = (lastRoapMessage, currentRoapMessage) => lastRoapMessage?.msg?.seq === currentRoapMessage.msg.seq && lastRoapMessage?.msg?.messageType === currentRoapMessage.msg.messageType;
```
Comment (marcin-bazyl) [inline packages/node_modules/@webex/plugin-meetings/src/roap/handler.js:37]: I think the function name should change to compareWithLastRoapMessage() or something like that as it is comparing any message type not just offer

File: packages/node_modules/@webex/plugin-meetings/src/constants.js [hunk 2]
```diff
@@ -444,7 +444,8 @@ export const EVENT_TRIGGERS = {
   MEETINGS_NETWORK_DISCONNECTED: 'network:disconnected',
   MEETINGS_NETWORK_CONNECTED: 'network:connected',
   MEETING_SELF_LEFT: 'meeting:self:left',
-  NETWORK_QUALITY: 'network:quality'
+  NETWORK_QUALITY: 'network:quality',
+  MEDIA_SERVER_NEGOTIATED: 'media:serverNegotiated'
```
Comment (marcin-bazyl) [inline packages/node_modules/@webex/plugin-meetings/src/constants.js:448]: maybe simply "media:negotiated" would be better? it's just a suggestion, you can keep it as it is.

**Review by marcin-bazyl — APPROVED** [review]
I've made a couple of minor comments, but overall looks good.

**Comment by arun3528** [comment]
fixed review comments merging it

**Pull request merged**